DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1

```diff
@@ -22,18 +22,17 @@ function Get-TargetResource
         [ValidateNotNullOrEmpty()]
         [string]
         $Name,
-        [ValidateNotNullOrEmpty()]
         [string]
         $Params,
-        [ValidateNotNullOrEmpty()]
-        [string]
+        [string[]]
         $Version,
-        [ValidateNotNullOrEmpty()]
         [string]
         $Source
     )
 
     Write-Verbose -Message 'Start Get-TargetResource'
+    $env:Path = [Environment]::GetEnvironmentVariable('Path','Machine')
+    Write-Verbose -Message "Path variables: $env:Path"

```

Comment:
Can you explain why this is here? It doesn't appear to be used for anything other than to output a Verbose message.

Reply:
no, it has also different reason, update environment variables of current session. I have some problems with software usage after install, because dsc can't find this app. But during install app update system env variables with paths and internal use functionality, but not current session env.

Reply:
Changing the current session path isn't a good idea as it potentially overwrites something that has been set elsewhere deliberately. If you want to change the path then either add to the session path what you need to or overwrite it and return it back to it's original state after you've done with it what you need.